src/test/java/longtimenosee/logic/parser/FindPolicyCommandParserTest.java

```diff
@@ -0,0 +1,92 @@
+package longtimenosee.logic.parser;
+
+import static longtimenosee.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static longtimenosee.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import longtimenosee.logic.commands.FindPolicyCommand;
+import longtimenosee.model.policy.Company;
+import longtimenosee.model.policy.Coverage;
+import longtimenosee.model.policy.Title;
+import longtimenosee.model.policy.predicate.CompanyMatchesInputPredicate;
+import longtimenosee.model.policy.predicate.CoverageMatchesInputPredicate;
+import longtimenosee.model.policy.predicate.TitleContainsKeywordsPredicate;
+
+public class FindPolicyCommandParserTest {
```

Comment:
Generally good coverage of most SUTs!

It's great that you followed the textbook convention of not having multiple invalid inputs inside each test case.